pkg/crc/machine/machine.go

```diff
@@ -159,12 +165,23 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error getting the state for host: %v", err)
 		}
 		if vmState == state.Running {
+			openshiftVersion, err := crcBundleMetadata.GetOpenshiftVersion()
+			if err != nil {
+				logging.Infof("A CodeReady Containers VM is already running")
+			} else {
+				logging.Infof("A CodeReady Containers VM for OpenShift %s is already running", openshiftVersion)
+			}
 			result.Status = vmState.String()
 			return *result, nil
 		}
 
 		if vmState != state.Running {
-			logging.Infof("Starting stopped VM ...")
+			openshiftVersion, err := crcBundleMetadata.GetOpenshiftVersion()
+			if err != nil {
+				logging.Infof("Starting VM ...")
+			} else {
+				logging.Infof("Starting VM for OpenShift %s...", openshiftVersion)
```

Comment:
Why drop the "stopped" from here?

Reply:
There is now a different log when the VM is already running, In "Starting VM" vs "Starting stopped VM", I think the stopped makes the sentence more complicated to read without really adding useful information.

Reply:
In this case using `Restarting` could indicate more meaning.